src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2711,7 +2727,7 @@ private void recordRelocation(void* location, void* target, ushort fRelocType, u
 
             TargetArchitecture targetArchitecture = _compilation.TypeSystemContext.Target.Architecture;
 
-            if (locationBlock != BlockType.Code)
+            if (locationBlock != BlockType.Code && locationBlock != BlockType.ROData)
```

Comment:
Let's move the `throw new NotImplementException` to the place where we're reading `locationBlock` to have this together and avoid the duplicated condition.

Reply:
Thank you. Fixed.